packages/rspack/tests/configCases/loader/context/index.js

```diff
@@ -0,0 +1,5 @@
+it("should be able to use a context with a loader", () => {
+	let a = "a";
+	let result = require(`raw-loader!./${a}`).default;
```

Comment:
There's no requirement to utilise ﻿`raw-loader` in this. I suggest creating a `﻿loader.js` file with the following code:

```js
module.exports = (content) => content;
```

and then replacing ﻿`require(raw-loader!./${a}﻿)` with `﻿require(./loader.js!./${a}﻿)`.

Reply:
IMO, using raw-loader can help confirm if the loader is functioning as expected. The `loader.js` file appears to return the original content, making it challenging to determine whether `loader.js` is working.

Reply:
To test if the loader is working, It's not necessary to use an additional loader.
Even a `content.replace("1", "2")` is enough.

Reply:
> To test if the loader is working, It's not necessary to use an additional loader. Even a `content.replace("1", "2")` is enough.

get~